app/code/Magento/Catalog/Controller/Adminhtml/Product/Save.php

```diff
@@ -270,31 +282,23 @@ protected function copyToStores($data, $productId)
     }
 
     /**
-     * Get categoryLinkManagement in a backward compatible way.
+     * Get CategoryLinkManagementInterface
      *
      * @return \Magento\Catalog\Api\CategoryLinkManagementInterface
      */
     private function getCategoryLinkManagement()
     {
-        if (null === $this->categoryLinkManagement) {
-            $this->categoryLinkManagement = \Magento\Framework\App\ObjectManager::getInstance()
-                ->get(\Magento\Catalog\Api\CategoryLinkManagementInterface::class);
-        }
         return $this->categoryLinkManagement;
     }
 
     /**
-     * Get storeManager in a backward compatible way.
+     * Get StoreManagerInterface
      *
      * @return StoreManagerInterface
      * @deprecated 101.0.0
      */
     private function getStoreManager()
```

Comment:
Please, remove method and replace usages to `$this->storeManager`

Reply:
Fixed, thank you